korean/marshmallow.md

```diff
@@ -8,7 +8,7 @@
 
 [다시 잔다](sleep/marshmallow.md)?
 
-[이건 아니다 싶어 영어 버션으로 간다](../english/marshmallow.md)?
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.